allauth/templates/account/email/base_notification.txt

```diff
@@ -0,0 +1,18 @@
+{% extends "account/email/base_message.txt" %}
+{% load account %}
+{% load i18n %}
+
+{% block content %}{% autoescape off %}{% user_display user as user_display %}{% blocktrans with site_name=current_site.name %}Greetings {{ user_display }},
```

Comment:
The `base_message.txt` already contains a greeting, so doing so here again results in:

```
Hello from example.com!

Greetings user,

You are receiving this mail because the following change was made to your account:

Your password has been changed.
```

We need to drop this greeting here.

Reply:
Removed the greeting